pkg/investigations/chgm/util_test.go

```diff
@@ -44,4 +45,8 @@ func TestCreateEgressLS(t *testing.T) {
 
 	result := createEgressLS(blockedUrls)
 	assert.Equal(t, *expected, *result)
+
+	// OCM does not like the parenthesis, stop gap until we figure
+	assert.Equal(t, strings.Contains(result.Details, "("), false)
```

Comment:
I don't think the test could ever contain a parenthesis, as the `blockedUrls` doesn't contain them. The issue is that `blockedUrls` we pass above has them in its new format. Can we update the test accordingly?

Reply:
good catch ;)